site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -135,6 +135,10 @@ export const CreateWorkspacePageView: FC<
     })
 
   const isLoading = props.loadingTemplateSchema || props.loadingTemplates
+  // We only want to show schema that have redisplay_value equals true
+  const schemaToBeDisplayed = props.templateSchema?.filter(
+    (schema) => schema.redisplay_value,
+  )
```

Comment:
The linked bug ticket mentions that terraform values set to `sensitive` shouldn't appear in the UI. Does the `sensitive` key map to `redisplay_value` or is this something we have to check for separately?